tokio/src/io/mod.rs

```diff
@@ -126,7 +126,8 @@
 //! implement [`AsyncRead`] and [`AsyncWrite`].
 //!
 //! Note that the standard input / output APIs  **must** be used from the
-//! context of the Tokio runtime, as they require Tokio-specific features to function.
+//! context of the Tokio runtime, as they require Tokio-specific features to
+//! function. Calling these functions outside of a Tokio runtime will panic.
```

Comment:
How about “will result in a panic”?

Reply:
Hmm, is that clearer? I don't want to imply that this _returns_ a `Result`?